README.rst

```diff
@@ -0,0 +1,41 @@
+|BuildStatus|_ |TotalDownloads|_ |LatestStableVersion|_ |License|_ |TYPO3|_
+
+.. |BuildStatus| image:: https://github.com/FriendsOfTYPO3/extension_builder/workflows/tests/badge.svg
+   :alt: Build Status
+.. _BuildStatus: https://github.com/FriendsOfTYPO3/extension_builder/actions
+
+.. |TotalDownloads| image:: https://poser.pugx.org/friendsoftypo3/extension-builder/d/total.svg
+   :alt: Total Downloads
+.. _TotalDownloads: https://packagist.org/packages/friendsoftypo3/extension-builder
+
+.. |LatestStableVersion| image:: https://poser.pugx.org/friendsoftypo3/extension-builder/v/stable.svg
+   :alt: Latest Stable Version
+.. _LatestStableVersion: https://packagist.org/packages/friendsoftypo3/extension-builder
+
+.. |License| image:: https://poser.pugx.org/friendsoftypo3/extension-builder/license.svg
+   :alt: License
+.. _License: https://packagist.org/packages/friendsoftypo3/extension-builder
+
+.. |TYPO3| image:: https://img.shields.io/badge/TYPO3-11-orange.svg
+   :alt: TYPO3
+.. _TYPO3: https://get.typo3.org/version/11
+
+=====================================
+TYPO3 Extension ``extension_builder``
+=====================================
+
+The *Extension Builder* helps you to develop a TYPO3 extension based on the
+domain-driven MVC framework `Extbase <https://docs.typo3.org/m/typo3/book-extbasefluid/master/en-us/0-Introduction/Index.html>`__
```

Comment:
hier würde ich auch auf v10 verlinken

Reply:
Hab ich in den letzten commit `[DOCS] Adapt links and text to Extension Builder v10` integriert.